.github/workflows/make_release.yml

```diff
@@ -9,11 +9,13 @@ permissions:
   contents: read
 
 jobs:
-  build:
+  release:
+    if: github.repository == 'systemd/systemd' || github.repository == 'systemd/systemd-stable'
```

Comment:
Hmm, we don't have `${{` here. If it is optional, then we could also drop it in other places below.

Reply:
It's optional in case of `if` conditions but required elsewhere.

docs: [Workflow syntax for GitHub Actions](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idif)

> _When you use expressions in an `if` conditional, you may omit the expression syntax (`${{ }}`) because GitHub automatically evaluates the `if` conditional as an expression._